server/models/IceRaidModel.js

```diff
@@ -0,0 +1,31 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+// Define collection and schema for Items
+var iceRaidCollectionModel = new Schema({
+  iceRaidPost: {
+    type: String
+  },
+  iceRaidAddress: {
+    type: String
+},
+  iceRaidZipcode: {
+      type: Number
+  },
+  iceRaidCity: {
+    type: String
+  },
+  iceRaidState: {
+    type: String
+},
+  iceRaidDate: {
```

Comment:
You've got two fields for the date, date and time, and neither of them use the native `Date` storage. I'd recommend using just one field and https://docs.mongodb.com/manual/reference/bson-types/#date